onnxruntime/python/tools/quantization/onnx_model.py

```diff
@@ -369,3 +368,93 @@ def topological_sort(self):
         assert end == len(self.graph().node), "Graph is not a DAG"
         self.graph().ClearField("node")
         self.graph().node.extend(sorted_nodes)
+
+    def clean_initializers(self):
+        return ONNXModel.clean_initializers_helper(self.graph(), self.model)
+
+    @staticmethod
+    def clean_initializers_helper(graph, model):
```

Comment:
It is required that this function be a staticmethod? If not prefer making it a private function in the module. (prepend `_` to its name and move out from the class)

 https://google.github.io/styleguide/pyguide.html#217-function-and-method-decorators